tests/robustness/failpoints.go

```diff
@@ -528,3 +529,35 @@ func (f delayPeerNetworkFailpoint) Name() string {
 func (f delayPeerNetworkFailpoint) Available(config e2e.EtcdProcessClusterConfig, clus e2e.EtcdProcess) bool {
 	return config.ClusterSize > 1 && clus.PeerProxy() != nil
 }
+
+type noSpaceFailpoint struct {
+}
+
+func (f noSpaceFailpoint) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster) error {
+	member := clus.Procs[rand.Int()%len(clus.Procs)]
+	for member.IsRunning() {
+		lg.Info("Setting up gofailpoint", zap.String("failpoint", f.Name()))
+		err := member.Failpoints().Setup(ctx, f.Name(), "return")
+		if err != nil {
+			lg.Info("goFailpoint setup failed", zap.String("failpoint", f.Name()), zap.Error(err))
```

Comment:
We need to assert that failpoint was executed at least once. Please follow https://github.com/etcd-io/etcd/issues/14729 on how to do that.

Reply:
done, it's much easier than sleep testing